app/renderer/js/components/webview.js

```diff
@@ -4,12 +4,14 @@ const path = require('path');
 const fs = require('fs');
 
 const DomainUtil = require(__dirname + '/../utils/domain-util.js');
+const ConfigUtil = require(__dirname + '/../utils/config-util.js');
 const SystemUtil = require(__dirname + '/../utils/system-util.js');
 const LinkUtil = require(__dirname + '/../utils/link-util.js');
 const { shell, app } = require('electron').remote;
 
 const BaseComponent = require(__dirname + '/../components/base.js');
 
+const silentWebview = ConfigUtil.getConfigItem('silent');
```

Comment:
**isWebviewSilent**.

Reply:
Actually `shouldSilentWebview` would be better.

Reply:
Yeah, that would be better :)